tests/benchmarks/class-internal.rkt.out

```diff
@@ -2188,7 +2202,10 @@ last few projections.
                                   (null? augride-names)
                                   (null? final-names))]
          [no-new-fields? (null? public-field-names)]
-         [xappend (lambda (a b) (if (null? b) a (append a b)))])
+         [xappend (lambda (a b)
```

Comment:
This one, too. Though I feel it could go either way in this specific case (perhaps because the LHS is just a single identifier?)

Reply:
This one I would write on the next line too. In general, I strongly prefer adding linebreaks at the highest level of nesting first, even if that means more vertical space is used overall.